libs/core/src/lib/form/form-label/form-label.component.scss

```diff
@@ -1,6 +1,7 @@
 @import '~fundamental-styles/dist/form-label';
 
 $form-label-inline-help-placement-space: 0rem;
+$form-label-bottom-spacing: 0.125rem;
```

Comment:
Maybe this should be also reflected in fundamental-styles?

Reply:
Wrapper does not exist on fundamental side at all. It would be a breaking change with no value to it I think